doc-deploy-changelog/action.yml

```diff
@@ -238,17 +266,13 @@ runs:
         fi
 
     - name: "Create PR into main with CHANGELOG changes"
-      if: contains(env.UPDATED_CHANGELOG, 'True')
+      if: ${{ (inputs.update-main-branch != 'true' ) && (env.UPDATED_CHANGELOG == 'True') }}
```

Comment:
Would it be possible to update previous lines of code to use `'false'` and `'true'` for consistency (see lines 262, 265, 269 and 325)?

Reply:
Yeah, here you go:
- https://github.com/ansys/actions/blob/b3dd7a4335078c6a3726786cf5f8bbb35c366b96/doc-deploy-changelog/action.yml#L234
- https://github.com/ansys/actions/blob/b3dd7a4335078c6a3726786cf5f8bbb35c366b96/doc-deploy-changelog/action.yml#L237
- https://github.com/ansys/actions/blob/b3dd7a4335078c6a3726786cf5f8bbb35c366b96/doc-deploy-changelog/action.yml#L241 (this is the line pointed by the previous comment)
- https://github.com/ansys/actions/blob/b3dd7a4335078c6a3726786cf5f8bbb35c366b96/doc-deploy-changelog/action.yml#301

Reply:
Ahh.. now I understand.

You want to use ``false`` instead of ``False``. And ``true`` instead of ``True``, right?

Reply:
I thought you were referring to the ``if: `` statement condition check (i.e. checking something else)

Reply:
Yeah, let's do this @klmcadams

Reply:
I should have quoted with links, my bad !